bcipy/language/model/gpt2.py

```diff
@@ -23,45 +24,143 @@ def __init__(self, response_type, symbol_set, lm_path=None):
         super().__init__(response_type=response_type, symbol_set=symbol_set)
         self.model = None
         self.tokenizer = None
-        self.is_start_of_word = True
         self.vocab_size = 0
         self.idx_to_word = None
-        self.curr_word_predicted_prob = None
         self.lm_path = lm_path or "gpt2"
+
+        # Hard coding a unigram language model trained on ALS phrase dataset
```

Comment:
If this is specific to the population or phrases used, it would be better to load these weights or smooth parameters in some other way. This overfits this model to the current experiment phrases. How would this change if we used a new dataset. What is the training procedure?

Reply:
I think we can think of this as the same as loading a pretrained GPT2 language model -- we can certainly put the weights in some other files and load from there, but we still need to train offline and update those files, which is not much different from what we are doing now. For now I think it's okay to keep it as it is.